src/Propel/Generator/Schema/Dumper/XmlDumper.php

```diff
@@ -503,8 +503,10 @@ private function appendIndexNode(Index $index, \DOMNode $parentNode)
     /**
      * Appends the generated <unique> XML node to its parent node.
      *
-     * @param Unique   $unique     The Unique model instance
+     * @param Unique   $index
```

Comment:
No, it's not common practice. I prefer as many others to have it not unnecessary verbose. So please don't introduce now full class names everywhere.